test/acceptance/tools/cygnus.py

```diff
@@ -519,12 +553,12 @@ def create_table (self, resource_name, service_path, attributes_number, attribut
         if resource_name == WITH_MAX_LENGTH_ALLOWED:
```

Comment:
The set of variable that are not going to change makes harder to read the conditions :S